packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -1,7 +1,7 @@
 /** @jsx withSlots */
 import * as React from 'react';
 import { Text, KeytipData } from '@fluentui/react';
-import { withSlots } from '../../Foundation';
```

Comment:
can we just remove things which use foundation from experiments?

Reply:
I don't think so; lots of components in `experiments` are still using foundation:
![image](https://user-images.githubusercontent.com/1207059/95270714-ff32d900-07f0-11eb-8297-b1727c160c3b.png)